ultravox/tools/push_to_hub.py

```diff
@@ -27,6 +27,8 @@ class UploadToHubArgs:
     data_type: Optional[str] = None
     # Public or private (default)
     private: bool = True
+    # Push LLM weights even if they are not fine-tuned
+    force_push_llm_weights: bool = True
```

Comment:
Do we want to do this by default? I tend to think we should simply update our vllm fork to be smarter here rather than force all users to redownload the llama 3.1 weights.

Reply:
To be clear, I'm OK to move forward with this PR, but I was thinking this option should default to `False`.

Reply:
I was holding off on the PR for now to see if @petersalas can solve the issue on his side. If it works, I'd rather remove the option tbh.

Reply:
This plan sounds good to me.